src/google-maps/google-map/google-map.ts

```diff
@@ -17,28 +34,93 @@ import {ReplaySubject} from 'rxjs';
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: '<div class="map-container"></div>',
 })
-export class GoogleMap implements OnInit {
-  // Arbitrarily chosen default size
-  @Input() height = '500px';
-  @Input() width = '500px';
+export class GoogleMap implements OnInit, OnDestroy {
+  @Input()
```

Comment:
What's your rationale for making these all write-only?

Reply:
I much prefer the setter method over using ngOnChanges, which I will discuss in the other comment. If you're asking why I don't have getter methods for these, I'm concerned that users will expect to get the current configuration of the map rather than the entered inputs, so I plan to copy the Google Maps API https://developers.google.com/maps/documentation/javascript/reference/map and have separate methods that retrieve the current state of the map.

Reply:
Yeah, I meant the omission of getters. My only concern here is that our documentation tooling doesn't support expressing something as write-only (vs. `readonly`), so we'd have to update that.